package.json

```diff
@@ -54,45 +54,45 @@
     "@types/mjml": "^4.7.0",
     "@types/mjml-react": "^2.0.0",
     "@types/ms": "^0.7.31",
-    "@types/node": "^16.7.1",
+    "@types/node": "^16.7.8",
     "@types/node-fetch": "^2.5.12",
     "@types/react": "^17.0.19",
     "@types/react-dom": "^17.0.9",
     "@types/react-router-dom": "^5.1.8",
     "@types/react-test-renderer": "^17.0.1",
-    "@typescript-eslint/eslint-plugin": "^4.29.3",
-    "@typescript-eslint/parser": "^4.29.3",
+    "@typescript-eslint/eslint-plugin": "^4.30.0",
+    "@typescript-eslint/parser": "^4.30.0",
     "assets-webpack-plugin": "^7.1.1",
-    "axe-core": "^4.3.2",
-    "babel-jest": "^27.0.6",
+    "axe-core": "^4.3.3",
+    "babel-jest": "^27.1.0",
     "babel-loader": "^8.2.2",
     "babel-plugin-transform-class-properties": "^6.24.1",
     "chalk-rainbow": "^1.0.0",
     "chromatic": "^5.9.2",
     "copy-node-modules": "^1.1.1",
-    "core-js": "^3.16.2",
-    "cypress": "^8.3.0",
+    "core-js": "^3.16.4",
+    "cypress": "^8.3.1",
     "cypress-axe": "^0.13.0",
     "dotenv-webpack": "^7.0.3",
     "eslint": "^7.32.0",
     "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-functional": "^3.6.0",
+    "eslint-plugin-functional": "^3.7.0",
     "eslint-plugin-jsx-a11y": "^6.4.1",
-    "eslint-plugin-prettier": "^3.4.1",
-    "eslint-plugin-react": "^7.24.0",
+    "eslint-plugin-prettier": "^4.0.0",
```

Comment:
Ref: https://github.com/prettier/eslint-plugin-prettier/blob/master/CHANGELOG.md

Reply:
> Thanks for doing this!
> 
> One thing, can we break the typescript changes out into a separate PR? This will help keep the git history useful for future us.
> 
> Also, there's some work triggered around the logo by the Source upgrade so it would be good to keep that separate as well (for similar reasons).

that split sounds good to me, will let you know once I've done that